Source/MetroXHunter/Sequencer/LevelSequencerHandler.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Implemented by Corentin Paya
+ */
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "LevelSequencerHandler.generated.h"
+
+class ALevelSequenceActor;
+class ACineCameraActor;
+class USceneComponent;
+
+UCLASS()
+class METROXHUNTER_API ALevelSequencerHandler : public AActor
+{
+	GENERATED_BODY()
+
+public:
+	ALevelSequencerHandler();
+	
+	UPROPERTY( VisibleAnywhere, BlueprintReadWrite, Category = "Interactable" )
```

Comment:
I guess it isn't finished, but lack of a `public` keyword to split the functions of the variables